storage.bs

```diff
@@ -151,16 +151,26 @@ must be cleared in its entirety.
 <h2 id=persistence>Persistence permission</h2>
 
 A <a>box</a> can only be turned into a <a>persistent box</a> if the user (or user agent
-on behalf of the user) has granted <dfn>permission</dfn>.
+on behalf of the user) has granted permission to use the {{"persistent-storage"}} feature.
 
 <p class="note">When granted to an <a>origin</a>, the persistence permission can be used to protect
 storage from the user agent's clearing policies. The user agent cannot clear storage marked as
 persistent without involvement from the <a>origin</a> or user. This makes it particularly useful for
 resources the user needs to have available while offline or resources the user creates locally.
 
-<p class="XXX">We will eventually integrate with the Permissions API, with the identifier
-"<dfn export for=PermissionName type=enum-value><code>persistent-storage</code></dfn>", but since
-that specification is not in great shape at the moment that has not happened yet.
+The <dfn for="PermissionName" enum-value>"<code>persistent-storage</code>"</dfn> <a>powerful feature</a>'s
+permission-related flags, algorithms, and types are defaulted, except for:
+
+<dl>
+ <dt><a>permission state</a></dt>
+ <dd><code>{name: {{"persistent-storage"}}}</code>'s <a>permission state</a> must have the same
+ value for all <a>environment settings objects</a> with a given <a>origin</a>.</dd>
```

Comment:
This was the question we decided in w3c/permissions#95, w3c/permissions#96, and w3c/permissions#97, and the discussion at https://www.w3.org/2016/06/08-webappsec-minutes.html: do we define a permission store with particular keys and values, or do we leave permission results up to the browser, with maybe some extra constraints. @benfredwells and @martinthomson liked the non-store option, so that's what we went with. Sorry you weren't in that meeting. :(